grails-app/assets/javascripts/spApp/service/biocacheService.js

```diff
@@ -38,7 +38,7 @@
                  *      "ws": "https://biocache.ala.org.au"
                  *  }
                  * - fqs
-                 *  ["spatiallyValid:true"]
+                 *  ["geospatial_kosher:true"]
```

Comment:
I thought `spatiallyValid` replaced `geospatial_kosher`? Dave thought the latter was culturally insensitive...